### PR TITLE
Improves Page Not Found UX

### PR DESCRIPTION
**Pull request by pranshuchittora**
Recently I made some changes to the docs #9486 which included deleting one file. But that doc page has already been indexed by the search engines & when opening shows :point_down: 

screenshot from 2018-11-01 02-04-04

But there is not message or information for the user about what happened & what do to next.
So I made some changes to improve UX :+1:

### CODE REVIEW

File: www/src/pages/404.js [hunk 1]
```diff
@@ -8,6 +8,12 @@ class FourOhFour extends React.Component {
       <Layout location={this.props.location}>
         <Container>
           <h1>Page not found</h1>
+          <p>
+            Oops! The page you are looking for has been removed or relocated.
+          </p>
+          <a href="../">
```
Comment (LekoArts) [inline www/src/pages/404.js]: When linking to internal pages you should use gatsby-link. Also the `to` prop then would be `"/"` not `../"`

File: www/src/pages/404.js [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
 class FourOhFour extends React.Component {
   render() {
     return (
       <Layout location={this.props.location}>
         <Container>
           <h1>Page not found</h1>
+          <p>
+            Oops! The page you are looking for has been removed or relocated.
+          </p>
+          <Link to="./">
```
Comment (DSchau) [inline www/src/pages/404.js]:
```suggestion
          <Link to="/">
```

File: www/src/pages/404.js [hunk 3]
```diff
@@ -1,13 +1,19 @@
 import React from "react"
 import Container from "../components/container"
 import Layout from "../components/layout"
-
+import { Link } from "gatsby"
 class FourOhFour extends React.Component {
```
Comment (DSchau) [inline www/src/pages/404.js]:
```suggestion

class FourOhFour extends React.Component {
```

(adding a space!)

File: www/src/pages/404.js (hunk 2 above)
Comment (DSchau) [reply www/src/pages/404.js]: @pranshuchittora just need to accommodate this one, and I can get this merged!

File: www/src/pages/404.js [hunk 4]
```diff
@@ -1,13 +1,20 @@
...
   render() {
     return (
       <Layout location={this.props.location}>
         <Container>
           <h1>Page not found</h1>
+          <p>
+            Oops! The page you are looking for has been removed or relocated.
+          </p>
+          <Link to="/">
+            <p>Go Back</p>
```
Comment (KyleAMathews) [inline www/src/pages/404.js]: Why is this "go back" not "go to frontpage" or something equivalent? This doesn't take you back anywhere

**Pull request merged**